xml/MAIN.VMDP.xml

```diff
@@ -54,7 +54,40 @@
     to existing operating systems.
    </para>
   </sect2>
-
+  <sect2 xml:id="sec-vmdp-signature">
+   <title>VMDP signature</title>
+   <sect3 xml:id="sec-vmdp-signature-vmdp-win">
+    <title>VMDP-WIN-VERSION</title>
+    <para>
+     VMDP-WIN-VERSION contains the Microsoft WHQL signed sys and cat files for the
+     Windows Server versions of the drivers contained in vmdp-211015.  The Windows
```

Comment:
what is vmdp-211015 ? a file or a package release? i would highlight it at least with &lt;literal/>

Reply:
lets remove the reference, and sayt that this is the same package execpt for the signature